sdk/appcenter-push/src/main/java/com/microsoft/appcenter/push/Push.java

```diff
@@ -173,7 +173,12 @@ public static void checkLaunchedFromNotification(Activity activity, Intent inten
      * Sender ID of your project before starting the Push service.
      *
      * @param senderId sender ID of your project.
+     *
+     * @deprecated Starting Android P at release date and later all versions of Android after April 2019,
+     * Firebase SDK is required to use push. Please follow the migration guide at https://aka.ms/...
+     * TODO: Set migration guide link.
```

Comment:
You can use https://aka.ms/acfba, thanks in advance (content will be updated or moved, for now it's the reversed migration page).